app/javascript/css/application.css

```diff
@@ -14,4 +14,23 @@
  *= require_self
  */
 
-.hello { color: green; }
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
+
+.hello {
+    color: green;
+    & .world {
+        @apply inline-block;
+
+        @screen md {
+            @apply font-bold;
+        }
+    }
+}
+
+input {
+    &::placeholder {
+        color: #123456;
+    }
+}
```

Comment:
I've done this manually to test that auto-prefixes get added (they do). Any thoughts on how we could write a regression test to ensure this works (ie, we don't accidently lose the prefixing later down the line). (cc @SleeplessByte too)

Reply:
I safari still uses a lot of vendor tags, you could write a test on a flex-box enable page to look for that css vendor prefix.

- e.g. http://shouldiprefix.com/#flexbox

Reply:
I'm going to move this to a separate issue: https://github.com/exercism/v3-website/issues/123